### ISSUE TITLE
Translate <collect-first-setting> result

### ISSUE THREAD

**Issue by Gabo6480** [Unprotesting/Auto-Tune#78]
The <collect-first-setting> placeholder returns a text version of the "collect-first" setting, this is a direct replacement. However this can be undesired on servers that have been translated.

**Comment by Unprotesting** [comment]
True, I will write a configurable message in messages.yml for each collect-first setting so that people can change the language.

**Comment by Gabo6480** [comment]
![image](https://user-images.githubusercontent.com/60194781/182279091-26835716-4c7b-44ef-8c2f-ad9955cd3c7a.png)
Also, a setting to translate the name of the shop related interfaces would be nice
